test/suite/test_sweep04.py

```diff
@@ -0,0 +1,225 @@
+        # populate
+        r = suite_random()
+        for i in range(0, self.core_tables):
+            self.create_table(self.core_uri(i))
+
+        created_transient = 0    # running total of transients created
+        available_transient = 0  # the next available transient number
+
+        # Create the initial batch of transient tables
+        while created_transient < self.transient_tables:
+            self.create_table(self.transient_uri(created_transient))
+            created_transient += 1
+
+        # Open all the session we'll use in advance
+        sessions = []
+        for i in range(self.nsessions):
+            sessions.append(self.conn.open_session())
+
+        # We keep the dhandle counts for each time we get stats
+        dhandle_counts = []
+
+        # The big loop: For half the run, we are stressing by accessing both core tables
+        # and transient tables, and creating/dropping transient tables. The second half
+        # of the run, we stop creating/dropping tables and only access core tables
+        # just to see if all outstanding dhandles are swept.
+        maxloop = self.transient_table_max * 2
+        lasttime = time.time()
+        for loopcount in range(0, maxloop):
+            stressing = (created_transient < self.transient_table_max)
+            if loopcount % 100 == 0:
+                self.pr('{}/{}  stressing={}'.format(loopcount, maxloop, stressing))
+
+                # Make sure at least 3 seconds elapses between each 100 times through
+                # the loop, to give the various sweeps time to operate.
+                thistime = time.time()
+                delta = thistime - lasttime
+                if delta < 3.0:
+                    time.sleep(3.0 - delta)
+                lasttime = thistime
+
+            if stressing:
+                self.session.drop(self.transient_uri(available_transient), "force")
+                available_transient += 1
+                self.create_table(self.transient_uri(created_transient))
+                created_transient += 1
+
+            rand_session = sessions[r.rand_range(0, self.nsessions)]
+
+            # In the stress part of the run, some small fraction (given by ratio_examined) will
+            # look at the transient tables.  Looking at these rarely makes them candidates for
+            # closing by the connection sweep.
+            big = 1000000   # any large number works
+            if stressing and r.rand32() % big > big * self.ratio_examined:
+                # look at transient tables
+                t = r.rand_range(available_transient, available_transient + self.transient_tables - self.transient_examined)
+                self.examine(rand_session, self.transient_uri, t, self.transient_examined)
+            else:
+                # look at a core table
+                t = r.rand_range(0, self.core_tables)
+                self.examine(rand_session, self.core_uri, t, 1)
+
+            if loopcount % 100 == 99:
```

Comment:
Add a comment to indicate we're only gathering and processing the stats now, not checking them.

Reply:
Added.